### ISSUE TITLE
mon crashes on 'ceph osd perf'

### ISSUE THREAD

**Issue by abh** [rook/rook#1306]
It was working before, but now the active(?) mon consistently crashes when I run 'ceph osd mon'.

```
2017-12-13 12:11:03.097181 I | rook-ceph-mon12: 2017-12-13 12:11:03.096908 7f20461c3700  0 mon.rook-ceph-mon12@1(peon) e27 handle_command mon_command({"prefix": "osd perf"} v 0) v1
2017-12-13 12:11:03.097437 I | rook-ceph-mon12: 2017-12-13 12:11:03.096981 7f20461c3700  0 log_channel(audit) log [DBG] : from='client.? 10.2.5.46:0/3540947190' entity='client.admin' cmd=[{"prefix": "osd perf"}]: dispatch
2017-12-13 12:11:04.365924 I | rook-ceph-mon12: 2017-12-13 12:10:12.707183 7f2051bfff40 -1 mon.rook-ceph-mon12@-1(probing).mgr e26 Failed to load mgr commands: (2) No such file or directory
failed to run mon. failed to start mon: Failed to complete rook-ceph-mon12: signal: aborted (core dumped)
```

**Comment by galexrt** [comment]
@abh Taking a guess as I had a similar issue, do you have monitoring through the rook-api enabled?

**Comment by abh** [comment]
I didn’t change anything for that, so only if it’s the default... it was working for a little bit and then stopped, fwiw.

**Comment by jbw976** [comment]
I believe this is the same crash being seen in #1111.  It would be good to finish off that work soon to enable the prometheus metrics module, although I don't have a proper fix for the standby mgr instances showing as DOWN.  Perhaps that's an acceptable condition for the time being so that we can at least get those metrics going/enabled, as long as the Grafana dashboards are also working and showing all the other metrics correctly.

**Comment by galexrt** [comment]
I'm closing this as issue #1111 has been closed by merged PR #1154.
This should not appear/happen when using the Ceph MGR monitoring.
